### DISCUSSION TITLE
Bootloader reinstall

### DISCUSSION THREAD

**Discussion by pio2398** [siderolabs/talos#8162] [Q&A]
I installed Talos in a virtual machine and everything seemed fine. However, after a reboot, it turned out that the system was unable to boot from the disk. Probably the bootloader did not install properly.
Now I am using iso to start system and then do the upgrade with the desired image (upgrade is doing kexec as I understate so no real reboot).  Creating new VM with clean install probably should solve problem but I am using Longhorn so if it is not necessary I would not want to do it. Can I somehow force to reinstall bootloader on disk?

**Comment by smira** [comment]
If Talos booted up into Kubernetes, it installed itself. The reason it doesn't boot might be related to the VM settings of some sorts.

**Comment by pio2398** [comment]
Well, probably not. Maybe  something wrong happened after install but even refind is not able to find anything bootable
![image](https://github.com/siderolabs/talos/assets/977500/78875f5f-1599-4fee-bc83-d1930645f8ad)

 I think reinstall is only reasonable option now.
